.github/workflows/release.yml

```
@@ -0,0 +1,21 @@
+name: Release
+
+on: [workflow_dispatch]
+
+jobs:
+  nuget:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository_owner == 'flutter-tizen' }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '6.0.x'
+      - name: Build and push
+        run: |
+          cd embedding/csharp/Tizen.Flutter.Embedding
```

How about using `working-directory` property?
```suggestion
      - name: Build and push
        working-directory: embedding/csharp/Tizen.Flutter.Embedding
        run: |
```